morelesszen_starterkit/js/morelesszen_starterkit.js

```diff
@@ -32,6 +32,31 @@ Drupal.behaviors.mobilemenu.attach = function(context, settings) {
   }
 };
 
+Drupal.behaviors.showMore = {};
+Drupal.behaviors.showMore.attach = function(context, settings) {
+  // link for revealing more info
```

Comment:
Please follow the [coding standard](https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards#comments) here.